lib/search.js

```diff
@@ -98,6 +98,11 @@ function filter (data, args, notArgs) {
 }
 
 function stripData (data) {
+  // Sanitize the maintainers field to ensure that NPM repositories (Artifactory) that
+  // does not sanitize this won't break the search functionality.
+  if (typeof data.maintainers === 'string') {
```

Comment:
Instead of mutating this maybe just put the array version in a variable and use that below?